index.html

```diff
@@ -14,7 +14,7 @@
     <title>NES.css - NES-style CSS Framework</title>
 
     <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
-    <link href="https://unpkg.com/nes.css/css/nes.min.css" rel="stylesheet" />
+    <link href="./css/nes.min.css" rel="stylesheet" />
```

Comment:
You shouldn't submit this change, this is just for local development. We need the unpkg url for the github hosted pages.

Reply:
Oh yeah, definitely. Sorry about that. 🙇